changelog/seqera-enterprise/v25.1.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Seqera Enterprise v25.1
+date: 2025-04-09
+tags: [seqera enterprise]
+---
+
+Seqera Platform Enterprise version 25.1 introduces Studios GA and a number of bug fixes and performance enhancements.
+
+Studios is Seqera's in-platform tool for secure, on-demand, interactive data analysis using containers created from Seqera-managed container template images or  your own organization managed custom environments. You only pay for the compute your Studio sessions consume and the compute is adjacent to your data, significantly reducing data transfer costs and wasted time copying data from storage to analysis. This significantly reduces infrastructure management requirements, removes data silos, adheres to strict in-platform role-based access control, and lowers your operational costs. Learn more about Studios in our [documentation](../../platform/latest/studios/index.mdx)
```

Comment:
```suggestion
Studios is Seqera's in-platform tool for secure, on-demand, interactive data analysis using containers created from Seqera-managed container template images or your own organization-managed custom environments. You only pay for the compute your Studio sessions consume and the compute is adjacent to your data, significantly reducing data transfer costs and wasted time copying data from storage to analysis. 

This, in turn, significantly reduces infrastructure management requirements, removes data silos, adheres to strict in-platform role-based access control, and lowers your operational costs. Learn more about Studios in our [documentation](../../platform/latest/studios/index.mdx)
```

Reply:
@robnewman, I'm leaving both instances of "significantly" for now but will mull over a possible replacement for one of them today.